### PR TITLE
[MNOE-165] Does not permit to install a second app if the app is not multi_instantiable

### PR DESCRIPTION
**Pull request by x4d3**
See https://github.com/maestrano/maestrano-hub/pull/153

add multi_instantiable and subcategories to App model.

### CODE REVIEW

**Comment by x4d3** [comment]
Done.

File: api/app/views/mno_enterprise/jpi/v1/marketplace/_app.json.jbuilder [hunk 1]
```diff
@@ -8,6 +8,8 @@ json.is_star_ready app.star_ready?
 json.is_connec_ready app.connec_ready?
 json.is_coming_soon app.coming_soon?
 json.single_billing app.single_billing?
+json.multi_instantiable app.multi_instantiable
+json.subcategories app.subcategories
```
Comment (ouranos) [inline api/app/views/mno_enterprise/jpi/v1/marketplace/_app.json.jbuilder]: Do we need the subcategories here?
If not it's better to keep the response smaller.

Comment (x4d3) [inline api/app/views/mno_enterprise/jpi/v1/marketplace/_app.json.jbuilder]: yes we do. See: https://github.com/x4d3/mno-enterprise-angular/blob/e44298f3ae80534941c8c0a11a80dbf68fe57932/src/app/views/marketplace/marketplace-app.controller.coffee#L75

Comment (ouranos) [inline api/app/views/mno_enterprise/jpi/v1/marketplace/_app.json.jbuilder]: Fair enough :+1:

**Comment by ouranos** [comment]
To be merged once maestrano/maestrano-hub#153 is merged

**Pull request merged**